frame/contracts/src/lib.rs

```diff
@@ -1265,7 +1265,7 @@ impl<T: Config> Pallet<T> {
 			ContractInfoOf::<T>::get(&address).ok_or(ContractAccessError::DoesntExist)?;
 
 		let maybe_value = contract_info.read(
-			&StorageKey::<T>::try_from(key).map_err(|_| ContractAccessError::KeyDecodingFailed)?,
```

Comment:
I am having a hard time understanding how the original code can compile here
How can we can invoke try_from here 
```rust
StorageKey::<T>::try_from(key)
```
and which impl is being called (the one from FixSizeKey or  VarSizedKey)

Reply:
@athei can you enlighten me here?

Reply:
Ah never mind I missed the confusing use statement here
```rust
pub use crate::{
	exec::{Frame, VarSizedKey as StorageKey},
```